src/components/Form.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+export default function Form() {
+  return (
+    <div>
+      <h1>ADD NEW BOOK</h1>
+      <input type="text" placeholder="book title" />
```

Comment:
Please be sure that your form also has an input for author 🧐

Reply:
You are correct, thanks.